### ISSUE TITLE
Captioning for deaf users?

### ISSUE THREAD

**Issue by kyleplo** [LLK/scratch-gui#1153]
![image](https://user-images.githubusercontent.com/31634240/34500657-b3accc7a-efd9-11e7-8709-466bc8d41a75.png)
I think that for deaf users, there could be captions above the stage.

**Blocks the produce captions:**
- Say: displays "[Sprite]: [Text]" - ex. "Cat: Hello!"
- Think: displays "[Sprite] thinks: [Text]" - ex. "Cat thinks: Hmm..."
- Sound: displays "♫ [Sound name] ♫" - ex. "♫ Meow ♫"
- Drum: displays "♫ [Drum name] ♫" - ex. "♫ Snare Drum ♫"
- Note: displays "♫ [Instrument] music ♫" - ex. "♫ Piano music ♫"

**Comment by apple502j** [comment]
we don't have to do with Say and Think (they're shown by default)

**Comment by kyleplo** [comment]
@apple502j Yeah, I guess that would make sense.

**Comment by hedgehog125** [comment]
Good idea, but there may be problems if there're unnamed/confusingly named sounds in the project.

**Comment by kyleplo** [comment]
@quachtina96 Here are some things we could do to solve that:
- Replace -,_ and / with a space
- Convert camel case - see https://kyleplo.github.io/b/uncamelcase/
- Remove ".mp3" and ".wav" at the end
